cmd/ctr-remote/logger/logger.go

```diff
@@ -44,19 +45,23 @@ const (
 func Mount(mountPoint string, tarfile io.ReaderAt, monitor Monitor) (func() error, error) {
 	// Mount filesystem.
 	root := newRoot(tarfile, monitor)
-	conn := nodefs.NewFileSystemConnector(root, &nodefs.Options{
-		NegativeTimeout: 0,
-		AttrTimeout:     time.Second,
-		EntryTimeout:    time.Second,
-		Owner:           nil, // preserve owners.
+	timeSec := time.Second
+	rawFS := fusefs.NewNodeFS(root, &fusefs.Options{
+		AttrTimeout:     &timeSec,
+		EntryTimeout:    &timeSec,
+		NullPermissions: true,
+	})
+	if err := root.InitNodes(); err != nil {
+		return nil, errors.Wrap(err, "failed to init nodes")
+	}
+	server, err := fuse.NewServer(rawFS, mountPoint, &fuse.MountOptions{
+		AllowOther: true,             // allow users other than root&mounter to access fs
+		FsName:     "logger",         // name this filesystem as "logger"
```

Comment:
why?

Reply:
About `FsName` field? It may be unnecessary. I'll fix it.